From 4e1f209f21c3958e5a8d3a0646dccf9e429ca7e4 Mon Sep 17 00:00:00 2001
From: Konstantin Tokarev <annulen@yandex.ru>
Date: Tue, 22 Sep 2020 14:26:31 +0300
Subject: [PATCH] Enable RTTI for ProcessLauncherQt.cpp to prevent crash in
 QProcess

In Qt 5.15 QProcess uses typeid of its object, and in ProcessLauncherQt.cpp
we subclass it.
---
 Source/WebKit2/PlatformQt.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Source/WebKit2/PlatformQt.cmake b/Source/WebKit2/PlatformQt.cmake
index f233e3cd920e7..395c6d9fc7072 100644
--- a/Source/WebKit2/PlatformQt.cmake
+++ b/Source/WebKit2/PlatformQt.cmake
@@ -225,6 +225,8 @@ if (COMPILER_IS_GCC_OR_CLANG)
         UIProcess/API/qt/qquickwebpage.cpp
         UIProcess/API/qt/qquickwebview.cpp
         UIProcess/API/qt/qwebiconimageprovider.cpp
+
+        UIProcess/Launcher/qt/ProcessLauncherQt.cpp
     PROPERTIES
         COMPILE_FLAGS -frtti
     )
